src/SingleSlider.elm

```diff
@@ -51,6 +51,7 @@ type alias Model =
 type Msg
     = TrackClicked String
     | RangeChanged String Bool
+    | OnChange String
```

Comment:
Can't think of a better name for this.

Reply:
RangeChanged -> when the sliders value changes
OnChange -> when you left go, so the final value

OnChange?
OnDrop? maybe look at drag and drop or range slider slibraries

Reply:
See latest commit - I think the `Msg`s should be named similarly to the events that they are produced by.